### PR TITLE
Issue 2170: Cleaned up error message for NoNodeException for lostBook…

### PR DESCRIPTION
**Pull request by Ghatage**
…ieRecoveryDelay when Auditor is not initialized.

### Motivation

If Auditor has not been initialized, running `lostBookieRecoveryDelay` from bookie shell will result in a stack trace and `NoNodeException`.

### Changes

Added a catch clause which:
* Cleans up stack trace
* Explains that the Auditor is not up yet, which means the znode for `lostbookierecoverydelay` has not been created yet.
* Prints exception

Master Issue: #2170

### CODE REVIEW

**Review by eolivelli — APPROVED** [review]
LGTM 

That stackteace was very annoying. Thanks

**Comment by eolivelli** [comment]
@Ghatage is it generally better to not merge your own pull requests.
Recently I committed my own patches after waiting the approval of other committers but that was because we haven't many active committers who had time to merge.

Now that you are onboard I see that we will move at a faster pace

**Comment by Ghatage** [comment]
> @Ghatage is it generally better to not merge your own pull requests

Understood!

**Pull request merged**
